src/processor/FHIRProcessor.ts

```diff
@@ -67,13 +68,36 @@ export class FHIRProcessor {
     return config;
   }
 
+  // Outputs a counter for each resource type being processed
+  outputCount(fhirArr: WildFHIR[], index: number, type: string) {
+    if (index == fhirArr.length - 1) {
+      // We want the logger to overwite the previous count
+      process.stdout.write('\x1b[A\r');
+      let finalMessage = `Processed ${fhirArr.length} ${type}`;
+      if (fhirArr.length === 1) {
+        logger.info(finalMessage.padEnd(45));
+        return;
+      } else {
+        finalMessage = finalMessage.concat('s');
+        logger.info(finalMessage.padEnd(45));
+        return;
+      }
+    } else if ((index + 1) % 5 == 0) {
+      // We want the logger to overwite the previous count
+      process.stdout.write('\x1b[A\r');
+      logger.info(`Processed ${index + 1} of ${fhirArr.length} ${type}s...\r`);
+    }
+  }
+
   process(config: ExportableConfiguration): Package {
     const resources = new Package();
     const igForConfig =
       this.lake.getAllImplementationGuides().find(doc => doc.path === this.igPath) ??
       this.lake.getAllImplementationGuides()[0];
     resources.add(config);
-    this.lake.getAllStructureDefinitions().forEach(wild => {
+    const structureDefs = this.lake.getAllStructureDefinitions();
+    logger.info('Processing StructureDefinitions...');
```

Comment:
This applies to all the messages like this, but since these messages are logged before we know that there are `StructureDefinitions`, we get something that looks like this: 
![image](https://user-images.githubusercontent.com/52747882/105710893-be68e800-5ee5-11eb-9a14-a7f718c8b338.png)
This is a case where my input had 1 SD and no other resources, but we still get the messages saying that we are `Processing CodeSystems...` and so on. I think we just want to only log these messages when the array of relevant resources is not empty.